funnel/templates/schedule_edit.html.jinja2

```diff
@@ -71,7 +71,7 @@
   </div>
   <div id="popup" class="modal session-modal">
     <div class="modal__header">
-      <a class="modal__close" href="javascript:void(0);" aria-label="close" rel="modal:close"><i class="material-icons" aria-hidden="true">clear</i></a>
+      <a class="modal__close" href="javascript:void(0);" aria-label="{% trans %}close{% endtrans %}" rel="modal:close"><i class="material-icons" aria-hidden="true">clear</i></a>
```

Comment:
Sentence case. Also, `rel="modal:close"` is new here. Are other places missing it?